apps/github/src/app.ts

```diff
@@ -0,0 +1,44 @@
+import AskBuddieBot from 'src/libs/askbuddiebot';
+import express from 'express';
+
+const app = express();
+const port = 3000;
+
+const askBuddieBot = AskBuddieBot.getInstance();
+
+app.use(express.json());
+
+app.post('/payload', async (req, res) => {
+    const payload = req.body as Payload;
+
+    // get event name and action from the request
+    const event = req.headers['x-github-event'] ?? null;
+
+    if (event === 'ping') return res.status(200).send('OK');
+
+    if (event === null) return res.status(400).send('Unknown event.');
+
+    const eventName = `${event}.${payload.action}`;
+
+    // find the registered event in the bot
+    const botEvent = askBuddieBot.getEvent(eventName);
+
+    // check if the event is from registered repository
+    if (
+        !askBuddieBot.isValidRepository(
+            (payload.repository?.name as string) ?? ''
```

Comment:
This doesn't work?

```suggestion
            payload.repository?.name ?? ''
```

Reply:
![image](https://user-images.githubusercontent.com/44132059/213743001-fad29b07-0e17-4473-8ea9-4f6df5a4d87e.png)
i get this error without typing it as string.